dev-support/docker/Dockerfile

```diff
@@ -89,7 +89,7 @@ RUN curl --location --fail --silent --show-error --output /tmp/hadolint "${HADOL
   echo "${HADOLINT_SHA256} */tmp/hadolint" | sha256sum -c -
 
 FROM BASE_IMAGE AS MAVEN_DOWNLOAD_IMAGE
-ENV MAVEN_VERSION='3.5.4'
+ENV MAVEN_VERSION='3.6.3'
 ENV MAVEN_URL "https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz"
 ENV MAVEN_SHA256 'ce50b1c91364cb77efe3776f756a6d92b76d9038b0a0782f7d53acf1e997a14d'
```

Comment:
Need to also update the artifact checksums.

Reply:
Thanks